cimr/processor/utils.py

```diff
@@ -90,3 +92,44 @@ def add_line_in_log():
     """Add an arbitrary divider to the log output for readability."""
     logging.info(' ' + '-' * 60 + '\n')
 
+
+def remove_palindromic(df):
+    """Removing palindromic variants from processed data."""
+    df = df.copy()
+    logging.debug(f'{df.head(2)}')
+    logging.debug(f'{df.columns}')
+    if 'ref' in df.columns:
+        df['g'] = df['alt'].str.upper() + df['ref'].str.upper()
+    elif 'effect_allele' in df.columns:
+        df['g'] = df['effect_allele'].str.upper() + df['non_effect_allele'].str.upper()
+    else:
+        logging.error(f' alleles need to be indicated.')
+        sys.exit(1)
+
+    df = df.loc[~((df.g == 'AT') | (df.g == 'TA') | (df.g == 'CG') | (df.g == 'GC'))]
```

Comment:
Bitwise operators are tricky. Can you write some comments here to explain exactly what this line is trying to do?